src/runmode-af-packet.c

```diff
@@ -195,7 +196,11 @@ static void *ParseAFPConfig(const char *iface)
             if (strcmp(threadsstr, "auto") == 0) {
                 aconf->threads = 0;
             } else {
-                aconf->threads = atoi(threadsstr);
+                if (ByteExtractStringInt32(&aconf->threads, 10, 0, (const char *)threadsstr) < 0) {
+                    SCLogError(SC_ERR_INVALID_VALUE, "Invalid number of "
```

Comment:
Should this be a warning ? (as we seem to go on)

Reply:
Yes, this should be. My fault.